packages/mobile/src/paymentRequest/OutgoingPaymentRequestListItem.tsx

```diff
@@ -26,14 +25,16 @@ interface OwnProps {
   updatePaymentRequestNotified: typeof updatePaymentRequestNotified
 }
 
+const avatarSize = 40
+
 type Props = OwnProps & WithNamespaces
 
 export class OutgoingPaymentRequestListItem extends React.Component<Props> {
   onRemind = () => {
-    const { id } = this.props
+    const { id, t } = this.props
     this.props.updatePaymentRequestNotified(id.toString(), false)
     CeloAnalytics.track(CustomEventNames.outgoing_request_payment_remind)
-    this.onFinalized()
+    Logger.showMessage(t('sendFlow7:requestSent'))
```

Comment:
Note: just so you know, these are not displayed on iOS for now since there's not equivalent to `ToastAndroid`.